packages/client/test/rpc/eth/getTransactionCount.spec.ts

```diff
@@ -74,15 +74,34 @@ describe(method, () => {
     assert.equal(res.result, `0x0`, 'should return 0x0 for nonexistent account')
   }, 40000)
 
-  it('call with unsupported block argument', async () => {
+  it('call with pending block argument', async () => {
     const blockchain = await Blockchain.create()
 
     const client = await createClient({ blockchain, includeVM: true })
     const manager = createManager(client)
+    const service = client.services.find((s) => s.name === 'eth') as FullEthereumService
     const rpc = getRpcClient(startRPC(manager.getMethods()))
 
-    const res = await rpc.request(method, ['0xccfd725760a68823ff1e062f4cc97e1360e8d997', 'pending'])
```

Comment:
Not sure, maybe we want to keep this test in it but now instead of using `pending` we use some invalid tag like `unsupported` or something.

Reply:
Meh I guess this is tested in the `getBlock` anyways